src/System.Management.Automation/commands/utility/FormatAndOutput/common/DefaultFormatters/DotNetTypes_format_ps1xml.cs

```diff
@@ -533,7 +533,6 @@ private static IEnumerable<FormatViewDefinition> ViewsOf_System_Diagnostics_Proc
         {
             yield return new FormatViewDefinition("process",
                 TableControl.Create()
-                    .AddHeader(Alignment.Right, label: "Handles", width: 7)
```

Comment:
Even on Full, I don't see value to the end user to know the number of handles to a process.  If they need it then they can inspect the object.